docs/README.md

```diff
@@ -4,38 +4,44 @@
 
 If you are a developer using Fn through the API, this section is for you.
 
-* [Quickstart](https://github.com/fnproject/fn)
-* [Usage](usage.md)
-* [Writing functions](writing.md)
-* [Testing functions](testing.md)
-* [Hot functions](hot-functions.md)
+* :star: [Quickstart](https://github.com/fnproject/fn)
+* :star: [Hot functions](hot-functions.md)
```

Comment:
Why are these starred?

Reply:
Trying to pin important topics to the top of the list. So I thought an icon might help identify them. I can use another icon or remove the emoji altogether. Let me know what you think.